tools/storage-usage-delta-calculator/src/main.rs

```diff
@@ -13,7 +13,10 @@ use tracing::debug;
 /// to get it
 fn main() -> std::io::Result<()> {
     let env_filter = near_o11y::EnvFilterBuilder::from_env().verbose(Some("")).finish().unwrap();
-    let _subscriber = near_o11y::default_subscriber(env_filter, &ColorOutput::Auto).global();
+    let runtime = tokio::runtime::Runtime::new().unwrap();
```

Comment:
For these examples it might be easier to use `#[tokio::main]` attribute as exemplified [here](https://docs.rs/tokio/latest/tokio/#examples).

Reply:
Thanks for spotting this.
Other 3 examples actually do actually drive runtime by creating actix or calling `block_on`.